content/docs/welcome-guide/setup/installing-linux.md

```diff
@@ -23,6 +23,15 @@ The following instructions assume that you have met all hardware and software re
     * `<path_to_deb_package>`: The path where you downloaded the deb package. Example: `~/Downloads`.
     * `<debian_package_name>`: The name of the package. Example: `O3DE_2111_1.deb`.
 
+    {{< known-issue >}}
+    If removing and installing the package, you may observe a warning like the following:
+        ```shell
+        W: Repository is broken: o3de:amd64 (= 21.11.1) has no Size information
+        ```
+    This is a [bug](https://salsa.debian.org/apt-team/apt/-/merge_requests/177) in apt that has already been fixed. The warning should go away once your current Linux distribution takes the fix.
+    The issue is a warning and does not affect installing or removing.
+    {{< /known-issue >}}
```

Comment:
There is a formatting issue to solve here. It's tricky because of the shortcode. It's extra tricky because the known-issue shortcode is currently broken. LOL. (I'm fixing it in a separate PR.)

I tested the following correction on my local build:

You'll have to remove the "/" in front of the shell code fencing below. I don't know how to escape the fencing when adding the suggestion (if GitHub even supports that).
```suggestion
    {{< known-issue link="https://salsa.debian.org/apt-team/apt/-/merge_requests/177" >}}
If removing and installing the package, you may observe a warning like the following:

/```shell
    W: Repository is broken: o3de:amd64 (= 21.11.1) has no Size information
/```

This is a bug in apt that has already been fixed. The warning should go away once your current Linux distribution takes the fix. The issue is a warning and does not affect installing or removing.
    {{< /known-issue >}}
```

Reply:
ok, corrected it